### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by sultanassi95**
#74
Editis for styling, routes, controllers, queries as requested in the previous issue,
build first by: @yahyaHB

### CODE REVIEW

File: public/css/singleCategory.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+.lower-div{
+  width: 100%
+}
+
+.jobt {
+  -moz-box-shadow: 0 2px 3px 0px rgba(0, 0, 0, 0.16);
+	-webkit-box-shadow: 0 2px 3px 0px rgba(0, 0, 0, 0.16);
+	box-shadow: 0 2px 3px 0px rgba(0, 0, 0, 0.16);
```
Comment (Walidmash) [inline public/css/singleCategory.css]: indentations

**Review by Walidmash — CHANGES_REQUESTED** [review]
only indentation problem

File: public/css/singleCategory.css (hunk 1 above)
Comment (sultanassi95) [reply public/css/singleCategory.css]: this is weird, I don't have these indentations on my editor...>!!!
:sob: 
@Walidmash

File: src/controllers/categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const { selectCategories } = require('../database/queries/queries');
+
+exports.get = (req, res) => {
+  selectCategories((err, result) => {
+    if (err) {
+      return res.status(500);
```
Comment (Walidmash) [inline src/controllers/categories.js]: use ```next(err);```

**Review by Walidmash — CHANGES_REQUESTED** [review]
error handling problem

**Review by Walidmash — APPROVED** [review]

**Pull request merged**